a-seed-from-nothing.sh

```diff
@@ -144,21 +139,24 @@ kolla_neutron_ml2_tenant_network_types:
 EOF
 fi
 
+# Set Environment variables for Kayobe dev scripts
+export KAYOBE_CONFIG_SOURCE_PATH=~/deployment/src/kayobe-config
+export KAYOBE_VENV_PATH=~/deployment/venvs/kayobe
+
 # Deploy hypervisor services.
-./dev/seed-hypervisor-deploy.sh
+~/deployment/src/kayobe/dev/seed-hypervisor-deploy.sh
 
 # Deploy a seed VM.
 # NOTE: This should work the first time because the packet configuration uses a
 # custom docker registry.  However, there are sometimes issues with Docker starting up on the seed (FIXME)
-if ! ./dev/seed-deploy.sh; then
+if ! ~/deployment/src/kayobe/dev/seed-deploy.sh; then
     # Deploy a seed VM. Should work this time.
-    ./dev/seed-deploy.sh
+    ~/deployment/src/kayobe/dev/seed-deploy.sh
 fi
 
 # Run TENKS
-cd ~/kayobe
-export TENKS_CONFIG_PATH=config/src/kayobe-config/tenks.yml
-./dev/tenks-deploy-overcloud.sh ./tenks
+export TENKS_CONFIG_PATH=~/deployment/src/kayobe-config/tenks.yml
+~/deployment/src/kayobe/dev/tenks-deploy-overcloud.sh ~/deployment/src/tenks
```

Comment:
Just want to double check you've tested this bit.
I've had issues in the past when running Tenks from any directory other than the kayobe root dir. I think there were a couple of different env vars that assumed paths

Reply:
Thanks for checking. There are three env vars you need to make sure it works, which are all exported before the script runs:
``` bash
export KAYOBE_CONFIG_SOURCE_PATH=~/deployment/src/kayobe-config
export KAYOBE_VENV_PATH=~/deployment/venvs/kayobe
export TENKS_CONFIG_PATH=~/deployment/src/kayobe-config/tenks.yml
```